Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -1 +1,271 @@
 
+## Notes for Flutter TDD Clean Architecture Course Videos
```

Comment:
@miguelmartins17 given that you have watched the first few videos of this course, 
you could give a quick summary at the start of the notes 
e.g: 
This video course takes the viewer through building an App that does ... [What doe the App do?]
The aim of the course is to build a basic App while following the best practices of TDD and clean architecture.